templates/app_runner.py

```diff
@@ -21,34 +28,37 @@ def _process_stage(stage_name, stage_value, config_subprocess_list):
         logging.info(line.rstrip())
 
     if process.returncode is not None:
-        logging.error(f"Error occurred in subprocess {stage_name}")
-        logging.error(process.returncode)
+        logging.info(f"Error occurred in subprocess {stage_name}")
+        logging.info(process.returncode)
         raise Exception(f"Error occurred in subprocess {stage_name}, with code {process.returncode}")
 
     end_time = time.time()
     logging.info(f'Stage {stage_name} completed in {end_time - start_time} seconds')
 
 
 def _upload_results(job_id: str):
-    with open('results_for_payload.json', 'r') as f:
-        results_for_payload = json.load(f)
-    AppRunnerUtils.upload_results(job_id, results_for_payload)
 
-    # if any additional files/artifacts to be uploaded
-    with open('results_for_upload.json', 'r') as f:
-        results_for_upload = json.load(f)
-    for element in results_for_upload:
+    #reads payload json if generated by code, otherwise gets from yaml
+    if (exists('/app/results_for_payload.json')) and (exists('/app/results_for_upload.json')):
```

Comment:
```suggestion
    if exists('/app/results_for_payload.json') and exists('/app/results_for_upload.json'):
```

python only requires parentheses in boolean expressions to set the boolean order